assets/queries/terraform/aws/unknown_port_exposed_to_internet/query.rego

```diff
@@ -30,7 +30,32 @@ CxPolicy[result] {
 
 	result := {
 		"documentId": input.document[i].id,
-		"searchKey": sprintf("aws_security_group[%s].ingress.from_port", [name]),
+		"searchKey": sprintf("aws_security_group[%s]", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("aws_security_group[%s].ingress.from_port is known", [name]),
+		"keyActualValue": sprintf("aws_security_group[%s].ingress.from_port is unknown and is exposed to the entire Internet", [name]),
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_security_group[name].ingress[x]
```

Comment:
Tip: you could have just one check for the ingress, where you call a function like *getIngressList* where you check the type of variable, if it is an array, return the variable, if it is an object, return an array with the object inside. This way you can reuse the code and save a couple of lines.